### PR TITLE
Remove experimental projects from 2.5 branch

### PR DESCRIPTION
**Pull request by asifsmohammed**
### Description
Remove experimental projects from 2.5 release branch
 
### Issues Resolved
Resolves #[Issue number to be closed when this PR is merged]
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has a documentation issue. Please link to it in this PR.
  - [ ] New functionality has javadoc added
- [x] Commits are signed with a real name per the DCO

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/data-prepper/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -131,15 +129,8 @@ include 'data-prepper-plugins:opensearch-source'
 include 'data-prepper-plugins:user-agent-processor'
 include 'data-prepper-plugins:in-memory-source-coordination-store'
 include 'data-prepper-plugins:aws-plugin-api'
-include 'data-prepper-plugins:aws-plugin'
```
Comment (dlvenable) [inline settings.gradle:134]: We need this plugin.

File: settings.gradle [hunk 2]
```diff
@@ -131,15 +129,8 @@ include 'data-prepper-plugins:opensearch-source'
...
 include 'data-prepper-plugins:obfuscate-processor'
 include 'data-prepper-plugins:parquet-codecs'
-include 'data-prepper-plugins:aws-sqs-common'
```
Comment (dlvenable) [inline settings.gradle:137]: I think we can keep this one. The `s3` source may already be using it.

**Review by dlvenable — CHANGES_REQUESTED** [review]

File: settings.gradle (hunk 2 above)
Comment (asifsmohammed) [reply settings.gradle:137]: I can keep it. But I checked and it's only being used in `sqs` source

**Review by dlvenable — APPROVED** [review]

File: settings.gradle (hunk 2 above)
Comment (dlvenable) [reply settings.gradle:137]: OK. I thought it was supposed to consolidate the SQS logic. Apparently it does no such thing.

**Review by engechas — APPROVED** [review]

**Pull request merged**